FirebaseSessions/Tests/Unit/SessionCoordinatorTests.swift

```diff
@@ -34,11 +34,38 @@ class SessionCoordinatorTests: XCTestCase {
       identifiers: identifiers,
       installations: installations,
       fireLogger: fireLogger,
-      sampler: sampler
+      sampler: sampler,
+      loggingEnabled: true
     )
     sampler.sessionSamplingRate = 1.0
   }
 
+  func test_attemptLoggingSessionStart_logsNothingWhenDisabled() throws {
```

Comment:
Can we create test cases where the loggingEnabled value is toggled? Basically wanted to validate the situation where disabling logging, followed by enabling logging makes sure we are sending events? and vice versa?

Reply:
All other test cases in this file should have `loggingEnabled` set to true

Reply:
Sure, that is true. I'm looking for test cases where the logging state is toggled at run-time. Eg: start with false, and then changes to true and then changes back to false and vice versa.